src/_includes/partials/helpus.njk

```diff
@@ -1,33 +1,35 @@
-<hr />
+<section class="py-400 mt-500 bt-sm text-secondary">
```

Comment:
You can remove the `text-secondary` class, non of the text elements in this partial are supposed to be using the secondary colour.

Reply:
I need the border to be gray, and that's the only way I found how to do it!

Reply:
Removed my changes

Reply:
I see, yeah I noticed that the gray border colour is not available as an option through the UF earlier as well. I made a note to add it! Could you potentially create a utility class for it in the CSS here? We will replace it once the UF class is available but having it here as custom code will help us figure out what we need to add to the UF.

Reply:
We can tackle this in another PR 👍